README.md

```diff
@@ -158,7 +158,7 @@ We have a badge! Link it to your live Storybook example.
 [![Storybook](https://cdn.jsdelivr.net/gh/storybookjs/brand@main/badge/badge-storybook.svg)](link to site)
 ```
 
-If you're looking for material to use in your presentation about storybook, like logo's video material and the colors we use etc, you can find all of that at our [brand repo](https://github.com/storybookjs/brand).
+If you're looking for material to use in your storybook presentation, such as logos, video material, and the colors we use, you can find it all on our  [brand repo](https://github.com/storybookjs/brand).
```

Comment:
@virtualwiz1 just one small item to address here, and we're good to go. If you wouldn't mind capitalizing storybook, I'd appreciate it.